### DISCUSSION TITLE
Cancelled ❌ Monthly Wallet and Dapp Meeting #27 [September] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#10553] [Wallet and (d)App Council]
# Brief scheduling note

## TLDR

This month's Wallet and Dapp meeting is cancelled ❌ The original date for the meeting was: Wednesday, 13 September (for future reference)

## Reason

There was relatively low attendance in the past few meetings, so I'm reconsidering the need for this meeting going forward.

## What next?

If you were interested in joining this meeting, please consider joining the bi-weekly "[Developer Office Hours](https://celoecosystem.com/office-hours)" instead. They are hosted by the great Developer Relations team at the Celo Foundation.

> Celo Foundation Developer Relations Team hosts Developer Office Hours bi-weekly.
> [Sign up here.](https://share.hsforms.com/15cP85PsIQcq91jdFx2OtcA53n4j)

<img width="740" alt="image" src="https://github.com/celo-org/celo-monorepo/assets/46296830/1510fa7c-9a86-4342-b02f-7155dea71f28">

**Comment by fbaud** [comment]
Hi @0xarthurxyz ,

There was low atendance, maybe because it was the summer season.

I also missed last meeting because the Celo Google Calendar mysteriously disappeared and I checked this discussion stream the day after the meeting.

While I understand it is hard to keep momentum over the months, I think a Wallet and Dapp Council is a good thing. Maybe it should be re-invented under another form to combat the natural wear and tear of time.

**Reply by vibern0** [reply]
Same here. I think it was very useful. And low attendance was because of the summer.
Sometimes I have calls at the same time, so last few have been a bit harder to attend, but still. I was going to join tomorrow.

**Reply by arthurgousset** [reply]
Thanks @fbaud and @obernardovieira for the kind words! 
I quite liked the meeting too, although I was always keen to have more people attend :) 
Let's reconvene at the next meeting and chat about ideas to make this meeting continue. 

To help get started, would you mind sharing specific examples of things you gained from attending the meeting?
Thanks!

**Reply by fbaud** [reply]
@0xarthurxyz ,

just to give some non exhaustive examples, I'd mention as positives of these discussions:
- learn the existence of libraries or other tools (e.g. RainbowKit)
- get new ideas by hearing other people views
- objectify perceptions of certain issues by hearing others' opinions (e.g. repetive discussions on WalletConnect, while sounding staled, always helped to re-assess subjective personal beliefs)

To give some directions on how these meetings could prove helpful in the future, I must mention that I worked lately extensively on OpenId Connect for Verifiable Credentials. I see a lot of parallels with the issues with face in the discussions between Dapps and Wallets. We may have to brainstorm on something like OpenId Connect for Crypto Payments in the future.

**Reply by arthurgousset** [reply]
Thanks for sharing @fbaud !
